tfx/experimental/templates/taxi/models/keras/model.py

```diff
@@ -205,11 +205,16 @@ def run_fn(fn_args):
     model = _build_keras_model(
         hidden_units=constants.HIDDEN_UNITS,
         learning_rate=constants.LEARNING_RATE)
-  # This log path might change in the future.
+
+   # TODO(b/158106209): deprecate logging to model artifact after next release
   log_dir = os.path.join(os.path.dirname(fn_args.serving_model_dir), 'logs')
+  if (fn_args.model_run_dir):
```

Comment:
did you check in tfx the release/old version (doesn't have model_run_dir defined when constructing TrainerFnArgs, not even set to None), will fn_args.model_run_dir be none or error?

Reply:
i think i remember seeing that this worked, but wasn't sure where. I've rewritten it to catch a key error instead

Reply:
you can test it print(fn_args.not_exist_name == None)

btw, I didn't see the change, did you commit?